bin/container.sh

```diff
@@ -175,39 +166,24 @@ start_cassandra() {
 }
 
 console_cassandra() {
-  wcip=$(docker inspect --format "{{ .NetworkSettings.Networks.${docker_network}.IPAddress }}" ${project}-cassandra)
-
-  docker run --net=${docker_network} -it --rm ${cassandra} cqlsh ${wcip} || \
-    usage "unable to run command: docker run --net=${docker_network} -it --rm ${cassandra} cqlsh ${wcip}" 1
+  docker exec -it ${project}-cassandra cqlsh || \
+    usage "unable to run command: docker exec -it ${project}-cassandra cqlsh" 1
 }
 
 start_mysql() {
   pwd=mypass
 
   start_docker
-  start_container ${project}-mysql ${mysql} "-p 3306:3306 -e MYSQL_ROOT_PASSWORD=${pwd}"
-
-  wmip=$(docker inspect --format "{{ .NetworkSettings.Networks.${docker_network}.IPAddress }}" ${project}-mysql)
-  sql=$(cat << EOF
-    create database if not exists ${project};
-    grant all privileges on ${project}.* to 'readwrite'@'localhost' identified by 'readwrite';
-    grant all on *.* to 'readwrite'@'%' identified by 'readwrite';
-    flush privileges;
-EOF
-)
-
-  docker run --net=${docker_network} -it --rm ${mysql} mysql -h${wmip} -P3306 -uroot -p${pwd} -e "${sql}" || \
-    usage "unable to run command: % docker run --net=${docker_network} -it --rm ${mysql} mysql -h${wmip} -P3306 -uroot -p${pwd} -e \"${sql}\"" 1
+  start_container ${project}-mysql ${mysql} "-p 3306:3306 -e MYSQL_ROOT_PASSWORD=${pwd} -e MYSQL_DATABASE=wasabi -e MYSQL_USER=readwrite -e MYSQL_PASSWORD=readwrite"
```

Comment:
really nice :) ... assuming the grants apply intrinsically?

nit: s/wasabi/${project}/